packages/cli/src/commands/run.ts

```diff
@@ -137,6 +137,7 @@ export async function run(packages: PackageSpecification[], options: RunOptions)
   }
 
   if (!signers.length) {
+    //TODO during writing tests I knew we never reach this point, signers is never empty because of this line signers = [provider.getSigner(addr)];
```

Comment:
@mohammadranjbarz Please remove this line and then we'll merge this as-is. Thanks!

Reply:
Done